apps/docs/src/pages/index.module.css

```
@@ -37,3 +40,147 @@
+  display: inline-flex;
+  margin-bottom: 0.5rem;
+
+  color: var(--ifm-color-primary);
+  font-size: 0.75rem;
+  font-weight: 700;
+  letter-spacing: 0.08em;
+  text-transform: uppercase;
+}
+
+.featureIntro {
+  margin: 0;
```

Stylelint (17.5.0)

[error] Unexpected duplicate "margin" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)